infra/README.md

```diff
@@ -0,0 +1,177 @@
+
+## AWS Authentication
+
+&nbsp;&nbsp;In order for Terraform to authenticate with your accounts you will need to configure your aws credentials using the AWS CLI or manually create your config and credentials file. If you need to manage multiple credentials or create named profiles for use with different environments you can add the `--profile` option.
+
+**~/.aws/credentials** (Linux & Mac) or **%USERPROFILE%\.aws\credentials** (Windows)
+
+### Examples:
+```
+$ aws configure
+AWS Access Key ID [None]: AKIAIOSFODNN7EXAMPLE
+AWS Secret Access Key [None]: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+Default region name [None]: us-east-2
+Default output format [None]: json
+```
+**Using the above command will create a [default] profile.**  
+```
+$ aws configure --profile dev
+AWS Access Key ID [None]: AKIAIOSFODNN7EXAMPLE
+AWS Secret Access Key [None]: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+Default region name [None]: us-east-2
+Default output format [None]: json
+```
+**Using the above command will create a [dev] profile.**  
+
+### References:
+
+- [Configuration basics][1]
+- [Named profiles for the AWS CLI][2]
+- [Configuration and credential file settings][3]
+
+[1]: https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html
+[2]: https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html
+[3]: https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html
+
+## Basic Terraform Commands 
+
+&nbsp;&nbsp;The `terraform init` command is used to initialize a working directory containing Terraform configuration files. This is the first command that should be run after writing a new Terraform configuration or cloning an existing one from version control. It is safe to run this command multiple times.
+
+&nbsp;&nbsp;The `terraform plan` command creates an execution plan, which lets you preview the changes that Terraform plans to make to your infrastructure. By default, when Terraform creates a plan it:
+
+- Reads the current state of any already-existing remote objects to make sure that the Terraform state is up-to-date.
+- Compares the current configuration to the prior state and noting any differences.
+- Proposes a set of change actions that should, if applied, make the remote objects match the configuration.
+
+&nbsp;&nbsp;The `terraform apply` command executes the actions proposed in a Terraform plan.
+
+<span style="color:red"> ---- DANGER ----</span>\
+In a non-development environment comment out a resource to destroy, destroy should only be used as a way to cleanup a development environment. e.g. a developers workspace after they are done with it.
+
+&nbsp;&nbsp; The `terraform destroy` command is a convenient way to destroy all remote objects managed by a particular Terraform configuration.\
+
+For more information about terraform commands follow the link below:
+
+- [Basic CLI Features](https://www.terraform.io/cli/commands)
+
+## Terraform Backend
+
```

Comment:
Since this is different than we've done on other projects like Mass PFML, I think we should add a little bit of writing here to just say that the approach is to store the terraform backend in AWS using its own resources.

Reply:
Will, I will also add the purpose is to separate the backend infra configuration from the application infra configuration.